src/redux/actions/paginationActions.js

```diff
@@ -0,0 +1,8 @@
+import { GET_CURRENT_PAGE } from 'redux/actions/types';
+
+export const getCurrentPage = (page = 1) => {
```

Comment:
Don't you think that this action actually setting current page not getting?

Reply:
True. :) I'm changing the name to a more appropriate.